### PR TITLE
Bug #1885 Correct comment grammar

### PR DESCRIPTION
**Pull request by harisankerPradeep**
Comment has been corrected

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/customtabs/CustomTabConfig.java [hunk 1]
```diff
@@ -163,7 +163,7 @@ public static CustomTabConfig parseCustomTabIntent(final @NonNull Context contex
         // Share is part of the default menu, so it's simplest just to toggle it off as necessary instead
         // of creating a fake menu item here, hence we keep this as  aboolean for now:
         final boolean showShareMenuItem = intent.getBooleanExtra(CustomTabsIntent.EXTRA_DEFAULT_SHARE_MENU_ITEM,
-                // Other than Chrome we will show the share menu by default. Note that currently the
+                // Unlike than Chrome we will show the share menu by default. Note that currently the
```
Comment (pocmo) [inline app/src/main/java/org/mozilla/focus/customtabs/CustomTabConfig.java]: Not a native speaker.. but in #1885 @mcomella says it should be "Unlike Chrome". This version says: "Unlike than Chrome".

Comment (harisankerPradeep) [reply app/src/main/java/org/mozilla/focus/customtabs/CustomTabConfig.java]: Sorry, my bad.

**Comment by harisankerPradeep** [comment]
So do you want me to create a new PR?

**Comment by pocmo** [comment]
You can use the same PR.

What you might want to do is an interactive rebase:
https://git-scm.com/book/id/v2/Git-Tools-Rewriting-History

It will allow you to meld multiple commits into a single one.

I also saw that you opened your PR from your `master` branch. That's no problem - but it saves you some headaches if you just open a branch for your patch and create a PR from there. :)

**Comment by mcomella** [comment]
> What you might want to do is an interactive rebase:
https://git-scm.com/book/id/v2/Git-Tools-Rewriting-History
> 
> It will allow you to meld multiple commits into a single one.

After this, you'll need to `git push -f` to overwrite your changes – be careful as this will **overwrite your changes** on the remote git repository.

> I also saw that you opened your PR from your master branch. That's no problem - but it saves you some headaches if you just open a branch for your patch and create a PR from there. :)

For example, it looks like a lot of the commits that are appearing in your PR are actually not yours, but from the master branch.

By the way, we might be able to help a little bit with git, if the documentation you find isn't helpful – you can find us on #focus on Mozilla IRC: https://wiki.mozilla.org/IRC

**Comment by mcomella** [comment]
> By the way, we might be able to help a little bit with git, if the documentation you find isn't helpful – you can find us on #focus on Mozilla IRC: https://wiki.mozilla.org/IRC

Oh, I see you already jumped on IRC – sorry we missed you! We're usually around working hours PST, Monday - Friday.

**Comment by harisankerPradeep** [comment]
What would be the best course of action now.? Should I create a new branch
and PR? Things do seem a bit complicated now.
Sorry if this is taking up too much time of yours.
Thanks in advance!

**Comment by mcomella** [comment]
> What would be the best course of action now.? Should I create a new branch and PR?

I think that would be best – it's hard to debug git issues asynchronously. For a high-level summary, your steps might look like:
```sh
git fetch <upstream> # get the latest changes
git checkout <upstream>/master -b <new-branch-name> # checkout latest changes and create a new branch
git cherry-pick <hash-for-your-first-commit> # grab the commit you want to save and put them on your new branch
```

Then you can push a new PR.

I oftentimes fallback to cherry-picking commits when I can't figure out how to fix a branch. :)

> Sorry if this is taking up too much time of yours.

No worries – we love others working with us on our projects and we love working with others on our projects. :)

**Comment by harisankerPradeep** [comment]
@mcomella  Well i guess i'll close this PR and open another one.
Thanks again!

**Comment by harisankerPradeep** [comment]
@mcomella I did try opening a new PR and doing the same, however, it couldn't pass the "Taskcluster" check.

**Pull request closed without merging**